### PR TITLE
Fix bug 1577482 - Show augmented original strings for Fluent

### PR DESCRIPTION
**Pull request by adngdb**
@mathjazz This time I (hopefully) made quite useful commits! Hope you'll enjoy them. :smile:

### CODE REVIEW

File: frontend/src/modules/fluentoriginal/components/FluentOriginalString.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            handleClickOnPlaceable={ props.handleClickOnPlaceable }
+        />;
+    }
+
+    // Complex, unsupported strings.
+    return <p className="original" onClick={ props.handleClickOnPlaceable }>
+        <WithPlaceablesForFluent>
+            { props.entity.original }
+        </WithPlaceablesForFluent>
+    </p>;
```
Comment (mathjazz) [inline frontend/src/modules/fluentoriginal/components/FluentOriginalString.js]: I guess this should be a `SourceStrings` component for consistency?

File: frontend/src/modules/entitydetails/components/OriginalStringProxy.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+export default function OriginalStringProxy(props: Props) {
+    if (props.entity.format === 'ftl') {
+        return <FluentOriginalString
+            entity={ props.entity }
+            handleClickOnPlaceable={ props.handleClickOnPlaceable }
+        />;
+    }
+
+    return <OriginalString
```
Comment (mathjazz) [inline frontend/src/modules/entitydetails/components/OriginalStringProxy.js]: I wonder if `GenericOriginalString` is a better name (for consistency with `*Editor` components)?

Comment (adngdb) [reply frontend/src/modules/entitydetails/components/OriginalStringProxy.js]: Yes, I absolutely agree!

File: frontend/src/modules/fluentoriginal/components/FluentOriginalString.js (hunk 1 above)
Comment (adngdb) [reply frontend/src/modules/fluentoriginal/components/FluentOriginalString.js]: Yeah, the arguments I had in my head against that don't really hold, now that I think about it again. Thanks!

**Pull request merged**